avatars/api.py

```diff
@@ -1,8 +1,9 @@
 # This file has been generated - DO NOT MODIFY
-# API Version : 0.3.5
+# API Version : 0.3.6
```

Comment:
We are at 0.3.7. We should add a check for rebase before releasing a new client.